src/Workspaces/Core/Portable/Rename/SymbolicRenameLocations.ReferenceProcessing.cs

```diff
@@ -121,6 +128,18 @@ static bool IsConstructorForType(ISymbol possibleConstructor, ISymbol possibleTy
             }
         }
 
+        private static bool IsPropertyGeneratedFromPrimaryConstructorParameter(
+            ISymbol propertySymbol, ISymbol parameterSymbol, CancellationToken cancellationToken)
+        {
+            if (parameterSymbol is IParameterSymbol parameter)
+            {
+                var generatedPropertySymbol = parameter.GetAssociatedSynthesizedRecordProperty(cancellationToken);
+                return propertySymbol.Equals(generatedPropertySymbol);
+            }
+
+            return false;
```

Comment:
this doesn't seem like the right fix to me.  Instead, shoudln't we be cascading between these members in the FIndReferences code.  Then, all these references should be found, and rename will update them autoamtically without having to think about it.

Reply:
FindReference code already cascades these. See:
https://github.com/dotnet/roslyn/blob/6a1c377d0562781e56daf66ee4fa9d2e0c9702a0/src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/PropertySymbolReferenceFinder.cs#L70
https://github.com/dotnet/roslyn/blob/6a1c377d0562781e56daf66ee4fa9d2e0c9702a0/src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/ParameterSymbolReferenceFinder.cs#L68

And the rename engine gets the correct symbol and locations [here](https://github.com/dotnet/roslyn/blob/6a1c377d0562781e56daf66ee4fa9d2e0c9702a0/src/Workspaces/Core/Portable/Rename/SymbolicRenameLocations.cs#L131).

But they are filtered out by this function.

Reply:
e.g. `ShouldIncludeSymbolAsync` in this file.

Reply:
oh.  niteresting.  thanks for the clarification.